package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "eslint-config-txo-typescript-react",
+  "version": "1.0.0",
+  "description": "Technology Studio - Eslint typescript react config",
+  "main": "src/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/technology-studio/eslint-config-txo-typescript-react.git"
+  },
+  "author": {
+    "name": "Technology Studio",
+    "email": "npm@technologystudio.sk",
+    "url": "https://github.com/technology-studio/"
+  },
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "lint": "eslint --format=node_modules/eslint-formatter-pretty --ext .jsx,.js,.ts,.tsx .",
+    "fixcode": "eslint --format=node_modules/eslint-formatter-pretty --ext .jsx,.js,.ts,.tsx . --fix",
+    "git-hook": "yarn -s lint-staged",
+    "test": "yarn lint",
+    "prepare": "husky install",
+    "sanity": "yarn lint && tsc --noEmit && yarn outdated && echo 'success'",
+    "semantic-release": "semantic-release"
+  },
+  "dependencies": {
+    "eslint-config-txo-typescript": "^3.1.24"
+  },
+  "peerDependencies": {
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-redux-saga": "^1.3.2"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^17.1.2",
+    "@commitlint/config-conventional": "^17.1.0",
+    "@semantic-release/changelog": "^6.0.1",
+    "@semantic-release/git": "^10.0.1",
+    "@types/react": "^18.0.21",
+    "@typescript-eslint/eslint-plugin": "^5.38.1",
+    "@typescript-eslint/parser": "^5.38.1",
+    "commitizen": "^4.2.5",
+    "eslint": "^8.24.0",
+    "eslint-formatter-pretty": "^4.1.0",
+    "eslint-plugin-eslint-comments": "^3.2.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.0.4",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-n": "^15.3.0",
+    "eslint-plugin-promise": "^6.0.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-redux-saga": "^1.3.2",
+    "husky": "^8.0.1",
+    "jest": "^29.1.1",
```

Comment:
we don't need jest at all

Reply:
we need jest packages because `eslint-config-txo-typescript` uses it, it also throws unmet peer dependency error when jest is removed